lang/en.json

```diff
@@ -253,7 +253,8 @@
 "DND5E.AdvancementSaveButton": "Save Advancement",
 "DND5E.AdvancementScaleValueTitle": "Scale Value",
 "DND5E.AdvancementScaleValueHint": "A single value that changes as a class increases in level that is made available in roll formulas (such as a Monk's martial arts die).",
-"DND5E.AdvancementScaleValueIdentifierHint": "This scale value for the current level will be available using <strong>@scale.{class}.{identifier}</strong> in a formula.",
+"DND5E.AdvancementScaleValueIdentifierHint": "This scale value for the current level will be available in a formula using:<br><copyable-text>@scale.{class}.{identifier}</copyable-text>",
+"DND5E.AdvancementScaleValueIdentifierDiceHint": "- whole formula (<code>4d6</code>)<br><copyable-text>@scale.{class}.{identifier}.die</copyable-text> - whole die (<code>d6</code>)<br><copyable-text>@scale.{class}.{identifier}.number</copyable-text> - number of dice (<code>4</code>)<br><copyable-text>@scale.{class}.{identifier}.faces</copyable-text> - number of faces (<code>6</code>)",
```

Comment:
I don't know how localisation-friendly this is, considering how word order changes a lot. Maybe it's fine since it's a list, but we could probably just repeat the whole of `AdvancementScaleValueIdentifierHint` in `AdvancementScaleValueIdentifierDiceHint` to be safe.